client/components/PostQuestionModal.jsx

```diff
@@ -0,0 +1,148 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+
+const PopoverLightbox = styled.div`
+  background: #0F1111;
+  z-index: 1008;
+  // opacity: 0.5;
+  display: block;
+`;
+
+const PopoverWrapper = styled.div`
+  position: relative;
+  border-radius: 8px;
+  // border: 1px solid;
+  border-color: #D5D9D9;
+  box-shadow: 0 0 14px 0 rgb(15 17 17 / 50%);
+  width: 460px;
+  height: 276px;
+  top: 363px;
+  left: 400px;
+  position: fixed;
+  background: rgba(0, 0, 0, 0.6);
+  background-color: #fff;
+`;
+
+const Header = styled.header`
+  position: relative;
+  border-bottom: 1px solid #D5D9D9;
+  background-color: #F0F2F2;
+  border-radius: 8px 8px 0 0;
+  padding: 0 24px;
+  height: 56px;
+`;
+
+const H4 = styled.h4`
+  padding: 16px 0;
+  min-height: 56px;
+  margin-right: 28px;
+  line-height: 24px;
+  font-size: 16px;
+  font-weight: 700;
+`;
+
+const Content = styled.div`
+  height: auto;
+  background-color: #fff;
+  overflow-y: auto;
+  padding: 16px 24px;
+  text-align: left;
+  overflow-x: hidden;
+`;
+
+const Form = styled.form`
+  margin-bottom: 0!important;
```

Comment:
Just a question about what !important means in this context?

Reply:
Yes, the !important is for css specificity